src/python/doc/nerve_gic_complex_user.rst

```diff
@@ -9,21 +9,27 @@ Definition
 
 .. include:: nerve_gic_complex_sum.inc
 
-Visualizations of the simplicial complexes can be done with either
+We provide two classes for computing cover complexes: one is CoverComplex, which can compute Mapper, Nerve and Graph Induced complexes in a scikit-learn format, 
+the other is NGIComplex, which only computes Nerve and Graph Induced complexes, but is a bit more flexible for input types (it can read inputs from paths to files for instance,
+while CoverComplex needs the inputs to be stored in memory).
+  
+Both classes can print output files, which can then be visualized with either
 neato (from `graphviz <http://www.graphviz.org/>`_),
 `geomview <http://www.geomview.org/>`_,
 `KeplerMapper <https://github.com/MLWave/kepler-mapper>`_.
-Input point clouds are assumed to be OFF files (cf. `OFF file format <fileformats.html#off-file-format>`_).
+Input point clouds (when they are read from files in NGIComplex) are assumed to be stored in OFF files (cf. `OFF file format <fileformats.html#off-file-format>`_).
+In the following, we provide examples for NGIComplex, and we refer to `this notebook <https://github.com/MathieuCarriere/tda-tutorials/blob/mapper/Tuto-GUDHI-cover-complex.ipynb>`_ for examples of CoverComplex.
```

Comment:
The link for the notebook should be (when notebook will be integrated) https://github.com/GUDHI/TDA-tutorial/blob/master/Tuto-GUDHI-cover-complex.ipynb

Reply:
Thanks! Done.